micro_brewery/demo/purchase_order_line.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="purchase_order_line_10" model="purchase.order.line">
+        <field name="order_id" ref="purchase_order_2"/>
+        <field name="product_id" ref="product_product_72"/>
+        <field name="partner_id" ref="res_partner_20"/>
```

Comment:
Why partner is there? When you create the PO line you have to add a partner in line?